tools/challenge-helper-scripts/create-project.ts

```diff
@@ -198,15 +198,7 @@ void prompt([
   {
     name: 'block',
     message: 'What is the dashed name (in kebab-case) for this project?',
-    validate: (block: string) => {
-      if (!block.length) {
-        return 'please enter a dashed name';
-      }
-      if (/[^a-z0-9-]/.test(block)) {
-        return 'please use alphanumerical characters and kebab case';
-      }
-      return true;
-    },
+    validate: validateBlockName,
     filter: (block: string) => {
       return block.toLowerCase();
```

Comment:
```suggestion
      return block.toLowerCase().trim();
```
If we're allowing trailing whitespace to pass validation, I think we should remove it here.